toolbox/py_xnvme_pkg/Makefile

```diff
@@ -0,0 +1,123 @@
+#
+# This Makefile serves as convenient command-line auto-completion when working on xNVMe-ctypes
+#
+#
+TOOLBOX_DIR?=../
+PROJECT_NAME=xnvme
+XNVME_CAPI_HEADERS=$(shell ls ../../include/libxnvme* | grep -v util | grep -v libxnvmec)
+XNVME_SO_PATH=$(shell pkg-config xnvme --variable=libdir)/libxnvme-shared.so
+
+define default-help
+# invoke: 'make uninstall', 'make install'
+endef
+.PHONY: default
+default: uninstall install
+	@echo "## py: make default"
+	@echo "## py: make default [DONE]"
+
+define build-help
+# invoke: 'python3 setup.py build'
+endef
+.PHONY: build
+build:
+	@echo "## py: make build"
+	@python3 setup.py build
+	@echo "## py: make build [DONE]"
+
+define build-sdist-help
+# Create a source package (invoke: 'python3 setup.py sdist')
+endef
+.PHONY: build-sdist
+build-sdist:
+	@echo "## py: make build-sdist"
+	@python3 setup.py sdist
+	@echo "## py: make build-sdist [DONE]"
+
+define install-help
+# install for current user
+endef
+.PHONY: install
+install:
+	@echo "## py: make install"
+	@pip3 install . --user
+	@echo "## py: make install [DONE]"
+
+define uninstall-help
+# uninstall
+#
+# Prefix with 'sudo' when uninstalling a system-wide installation
+endef
+.PHONY: uninstall
+uninstall:
+	@echo "## py: make uninstall"
+	@pip3 uninstall ${PROJECT_NAME} --yes || echo "Cannot uninstall => That is OK"
```

Comment:
`setup.py` dictates the name to provide here. It's currently `py-xNVMe`, and not `xnvme` as `${PROJECT_NAME}` resolves to.

Reply:
Fixed.